README.rst

```diff
@@ -678,6 +678,25 @@ information by performing lookups on the OUI
 
 This feature is enabled by default.
 
+``OPENWISP_MONITORING_RETRY_OPTIONS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+--------------------------------------------------------------------------------------------------+
+| **type**:    |                                            ``dict``                                              |
++--------------+--------------------------------------------------------------------------------------------------+
+| **default**: | ``{'max_retries': None, 'retry_backoff': True, 'retry_backoff_max': 600, 'retry_jitter': True}`` |
++--------------+--------------------------------------------------------------------------------------------------+
```

Comment:
Please show the default value as it's done here for consistency and also for better readability:
https://github.com/openwisp/openwisp-firmware-upgrader/#openwisp_firmware_upgrader_retry_options